interfaces/navigateur/public/libs/Ext.ux/FileUploadField/FileUploadField.js

```diff
@@ -88,20 +88,20 @@ Ext.ux.form.FileUploadField = Ext.extend(Ext.form.TextField,  {
         this.fileInput.on({
             scope: this,
             mouseenter: function() {
-                this.button.addClass(['x-btn-over','x-btn-focus'])
+                this.button.addClass(['x-btn-over','x-btn-focus']);
             },
             mouseleave: function(){
-                this.button.removeClass(['x-btn-over','x-btn-focus','x-btn-click'])
+                this.button.removeClass(['x-btn-over','x-btn-focus','x-btn-click']);
             },
             mousedown: function(){
-                this.button.addClass('x-btn-click')
+                this.button.addClass('x-btn-click');
             },
             mouseup: function(){
-                this.button.removeClass(['x-btn-over','x-btn-focus','x-btn-click'])
+                this.button.removeClass(['x-btn-over','x-btn-focus','x-btn-click']);
             },
             change: function(){
                 var v = this.fileInput.dom.value;
-                this.setValue(v);
```

Comment:
Si on le laisse là:
Uncaught InvalidStateError: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string.
En plus le vrai path ne peut pas être connu du navigateur alors ça sert seulement à mettre: C:\fakepath\monfichier.extension